Misc/NEWS.d/next/Build/2019-04-02-01-59-26.bpo-36500.fyG6_U.rst

```diff
@@ -0,0 +1 @@
+Added an optional "regen" project to the Visual Studio solution that will regenerate all grammar, token, AST and opcode header files, autogenerated C and Python modules.
```

Comment:
Thanks for the PR @tonybaloney.

This news entry needs to be split into multiple lines, the maximum character width for reST is 80. 

```suggestion
Added an optional "regen" project to the Visual Studio solution that will
regenerate all grammar, token, AST and opcode header files, autogenerated C and
Python modules.
```

If possible, I would also recommend trying to make it a bit more succinct.